src/OSPSuite.Presentation/Presenters/Importer/ImporterPresenter.cs

```diff
@@ -184,7 +184,7 @@ private void loadSheetsFromDataPresenter(object sender, ImportSheetsEventArgs ar
          {
             {
                var eMessage = e.Message;
-               if (! (e is NanException || e is ErrorUnitException || e is MissingColumnException || e is PossibleUnsupportedSheetFormatException))
+               if (! (e is NanException || e is ErrorUnitException || e is MissingColumnException || e is PossibleUnsupportedSheetFormatException || e is EmptyDataSetsException))
```

Comment:
There is somethning that I dont' really like about this. Maybe we need to create another base class for those type of exception that we are catching explicitly.
Thoughts?

Reply:
Yes, @georgeDaskalakis and I discussed this already. The code on his PR #1204 is also involved and we decided to merge this and then solve it on a separate issue to avoid conflicts. Will this work for you @msevestre?

Reply:
ok. I am going to createa an issue then and merge

Reply:
https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/1218